addon/components/ember-table-cell.js

```diff
@@ -1,9 +1,15 @@
 import EmberTableBaseCell from './ember-table-base-cell';
-import { property } from '../utils/class';
+import { readOnly } from 'ember-decorators/object';
+import { alias } from 'ember-decorators/object/computed';
+import { className } from 'ember-decorators/component';
 
 import layout from '../templates/components/ember-table-cell';
 
 export default class EmberTableCell extends EmberTableBaseCell {
-  @property layout = layout;
-  @property classNameBindings = ['isFixed::et-td'];
+  layout = layout;
+
+  // We have to alias because the class name changes per base cell type
+  @className('', 'et-td')
+  @readOnly @alias('isFixed') _isFixed;
```

Comment:
`@className` binds the property to a class name, and we provide those class names and the arguments. We can do it directly to `isFixed` because of problems with how it works in ember-decorators, so we need to make an alias.

Reply:
Going to open up a bug report in ember-decorators, but this would be a breaking change so we’ll need to hold off for some time.